tools/read-dependencies

```diff
@@ -46,11 +47,21 @@ MAYBE_RELIABLE_YUM_INSTALL = [
     """
     error() { echo "$@" 1>&2; }
     configure_repos_for_proxy_use() {
-        grep -q "^proxy=" /etc/yum.conf || return 0
+        if [ -f /etc/yum.conf ]; then
+            CFG_FILE=/etc/yum.conf
+        fi
+        if [ -f /etc/dnf.conf ]; then
```

Comment:
Some fedora/41 images have `/etc/dnf.conf` without the `/etc/yum.conf` symlink present.